utils/resource/usage.go

```diff
@@ -102,6 +111,34 @@ func (m *manager) DiskUsage() (float64, float64) {
 	return m.readUsage, m.writeUsage
 }
 
+func (m *manager) DiskNumberRead(pid int) uint64 {
+	m.usageLock.RLock()
+	defer m.usageLock.RUnlock()
+
+	return m.processes[pid].read
+}
+
+func (m *manager) DiskReadBytes(pid int) uint64 {
+	m.usageLock.RLock()
+	defer m.usageLock.RUnlock()
+
+	return m.processes[pid].lastReadBytes
+}
+
+func (m *manager) DiskNumberWrite(pid int) uint64 {
+	m.usageLock.RLock()
+	defer m.usageLock.RUnlock()
+
+	return m.processes[pid].write
+}
+
+func (m *manager) DiskWriteBytes(pid int) uint64 {
+	m.usageLock.RLock()
+	defer m.usageLock.RUnlock()
+
+	return m.processes[pid].lastWriteBytes
+}
```

Comment:
I think these should just be removed.

1. They aren't used anywhere
2. They panic if a `pid` that isn't being currently tracked is requested
3. They grab the `usageLock.RLock` but they actually need the `processesLock` (which we would probably want to turn into an RLock if we were exposing information like this)

Reply:
updated